chapters/interfaces.adoc

```diff
@@ -3368,8 +3368,8 @@ primitives.
     The code:PrimitivePointIndicesEXT decoration must: be used only within
     the code:MeshEXT {ExecutionModel}
   * [[VUID-{refpage}-PrimitivePointIndicesEXT-07042]]
-    The code:PrimitivePointIndicesEXT decoration must: be used only with the
-    the code:OutputPoints {ExecutionMode}
+    The code:PrimitivePointIndicesEXT decoration must: not be used with the
```

Comment:
Wouldn't it be clearer to just say that if a builtin is used then the execution mode must: be equal to OutputPoints for example?

Reply:
I think that's what the spirit of "used only with" was trying to express.

Reply:
:100: that seems better to me too, will fix

Reply:
WG 2022-10-05: We had the same idea in discussion - if you list the things it must NOT be, then if we add more execution modes, you have to remember to update these lists. So it seems better to say what it MUST be. But we're willing to be convinced if there's something you're trying to do here that we are not understanding.